bin/improver-nbhood

```diff
@@ -63,6 +63,11 @@ def main():
                        'should be used. For example: 10,12,14 1,2,3 '
                        'where a lead time of 1 hour uses a radius of 10km, '
                        'a lead time of 2 hours uses a radius of 12km, etc.')
+    parser.add_argument('--ens_factor', metavar='ENS_FACTOR', type=float,
+                        default=1.0, required=False,
+                        help='The factor with which to multiple '
+                        'the adjustment to the radii_in_km for more than '
+                        'one ensemble member. Optional, defaults to 1.0')
```

Comment:
I've changed this but I've kept Optional, defaults to 1.0. in Utilities:adjust_nsize_for_ens it is not optional and has no default